doc/overview/build.md

```diff
@@ -0,0 +1,127 @@
+# Endless OS 7 build process
+
+This document gives an overview of the tools used to build Endless OS 7.
+
+For guidance on how to build EOS7, see: [`doc/howto/build.md`](doc/howto/build.md).
+
+## OSTree build stage
+
+This stage is defined in
+[eos-build-meta.git](https://github.com/endlessm/eos-build-meta/) using the BuildStream build tool.
+
+It uses some 3rd party BuildStream plugins, which are listed in `project.conf`.
+
+It uses elements from Freedesktop SDK and GNOME Build Metadata via the junction
+elements `freedesktop-sdk.bst` and `gnome-build-meta.bst`.
+
+More details on each of these concepts below.
+
+### BuildStream
+
+BuildStream's documentation is available at:
+<https://docs.buildstream.build/>.
+
+Note that BuildStream is a plugin-based tool. Source fetching is done via
+*source plugins*, and element builds are handled by *element plugins*.
+
+The core of BuildStream ships a small set of plugins. Their documentation
+is found in the BuildStream manual under
+["Reference > Plugin specific documentation"](https://docs.buildstream.build/master/core_plugins.html).
+
+### BuildStream Plugins
+
+The BuildStream Plugins project is a set of source and element plugins
+maintained by the BuildStream core team.
+
+Documentation for BuildStream Plugins is available at:
+<https://apache.github.io/buildstream-plugins/>.
+
+### BuildStream Community Plugins
+
+The BuildStream Community Plugins project is a set of 3rd party source and
+element plugins. Some of these are developed and maintained as part of the
+Freedesktop SDK project.
+
+Documentation for BuildStream Community Plugins is available at:
+<https://buildstream.gitlab.io/buildstream-plugins-community/>.
+
+### Freedesktop SDK
+
+Freedesktop SDK provides integration instructions for a Linux runtime,
+using components commonly used in many Linux distributions.
+
+Some documentation for Freedesktop SDK is available at:
+<https://freedesktop-sdk.gitlab.io/documentation/>.
+
+Freedesktop SDK defines elements for many open source components in the
+[components/ subdirectory](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/tree/master/elements/components).
```

Comment:
You've used the `verbatim` mode for file paths, I don't know if it's worth making it consistent or if it's different enough as you're referring to another repo.